tck/src/main/java/ee/jakarta/tck/data/standalone/persistence/Catalog.java

```diff
@@ -73,7 +73,7 @@ public interface Catalog extends DataRepository<Product, String> {
     int countBySurgePriceGreaterThanEqual(Double price);
 
     @Query("SELECT p FROM Product p WHERE (SIZE(p.departments) = ?1 AND p.price < ?2) ORDER BY p.name")
-    Streamable<Product> findByDepartmentCountAndPriceBelow(int numDepartments, double maxPrice);
+    Page<Product> findByDepartmentCountAndPriceBelow(int numDepartments, double maxPrice);
```

Comment:
Cannot be Page because it has no PageRequest
```suggestion
    List<Product> findByDepartmentCountAndPriceBelow(int numDepartments, double maxPrice);
```